contributing/getting-started-quick.md

```diff
@@ -0,0 +1,73 @@
+# About this guide
+
+This Quick Start guide is the TL;DR version of the longer [end-to-end guide](getting-started-e2e.md) for people who don't want/need a longer explanation.
+
+# One-time Setup
+
+* [Create a GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/) if you don't already have one
+* Set up [2 factor auth](https://help.github.com/articles/about-two-factor-authentication/) for your GitHub account
+
+* [Install and set up Git](https://help.github.com/articles/set-up-git/); in the "Authenticating" step of that page use SSH instead of HTTPS
+
+* Install [NodeJS](https://nodejs.org/)
+
+* Install [yarn](https://yarnpkg.com/en/docs/install)
+
+* Install Gulp by running `yarn global add gulp`
+
+* Add this line to your hosts file (`/etc/hosts` on Mac or Linux, `%SystemRoot%\System32\drivers\etc\hosts` on Windows):
+
+    ```127.0.0.1               ads.localhost iframe.localhost```
+
+* Fork the [amphtml repository](https://github.com/ampproject/amphtml) by clicking "Fork" in the Web UI.
+
+* Create your local repository: `git clone git@github.com:<your_username>/amphtml.git`
+* Add an alias: `git remote add upstream git@github.com:ampproject/amphtml.git`
+
+# Branch (do this each time you want a new branch)
+
+* Create the branch: `git branch --track <branch name> origin/master`
+* Go to the branch: `git checkout <branch name>`
+
+# Build AMP & run a local server
+* Make sure you have the latest packages (after you pull): `yarn`
+* Start the server: `gulp`
+* Access your server at [http://localhost:8000](http://localhost:8000).
+* Access your sample pages at [http://localhost:8000/examples](http://localhost:8000/examples).
+* Change the suffix for the examples from .html to .max.html to use your local JavaScript.
+
+# Test AMP
+* Run the tests: `gulp test`
+* Run a single test: `gulp test --files=<filename`
```

Comment:
I personally prefer the `describe.only` method because it allows switching the tested files without restarting `gulp test`. It is slightly harder to explain, though.